### ISSUE TITLE
Make IPMI OEM commands optional feature

### ISSUE THREAD

**Issue by Kostr** [openbmc/phosphor-pid-control#43]
Currently `phosphor-pid-control` requires `phosphor-host-ipmid` for the build, to support IPMI OEM command functionality ([ipmi.md](https://github.com/openbmc/phosphor-pid-control/blob/master/ipmi.md)).

It would be a nice to have this functionality as an optional feature opposed to the necessary requirement, since `phosphor-pid-control` can be used on the systems without IPMI stack.

**Comment by Krellan** [comment]
Agreed. A patch is welcome, to implement the feature of making IPMI optional.
